### PR TITLE
#488, #495: Update to Xcode8 / iOS 10

### PR DESCRIPTION
**Pull request by bonkey**
#488, #495: Update to Xcode8 / iOS 10

### CODE REVIEW

File: AtlasSDK/AtlasSDK/Utils/NSLocale.swift [hunk 1]
```diff
@@ -10,12 +10,17 @@ extension NSLocale {
         return self.objectForKey(key) as? String ?? ""
     }
 
-    var countryCode: String {
-        return self.stringForKey(NSLocaleCountryCode)
-    }
+    // countryCode and languageCode were introduced with iOS 10,
+    // but there's no compile-time check iOS version
```
Comment (hani-ibrahim) [inline AtlasSDK/AtlasSDK/Utils/NSLocale.swift]: This is not working `#available(iOS 1.0, *)` ?

Comment (bonkey) [inline AtlasSDK/AtlasSDK/Utils/NSLocale.swift]: Probably it would, but compiler error comes from Objective-C. Anyway I've simplified it.

**Comment by hani-ibrahim** [comment]
`The command "bundle exec ./calypso.rb xcode test 3" exited with 65.`

**Comment by bonkey** [comment]
👍

**Comment by bonkey** [comment]
👍

**Comment by hani-ibrahim** [comment]
👍

**Pull request merged**